test/getPeerDependenciesFromRegistry.test.ts

```diff
@@ -15,6 +15,17 @@ describe('getPeerDependenciesFromRegistry', function () {
     })
   })
 
+  it('single package with range', async () => {
+    await chalkInit()
+    const data = await getPeerDependenciesFromRegistry({ 'eslint-plugin-unused-imports': '^4' }, {})
```

Comment:
`getPeerDependenciesFromRegistry` is currently typed to take an exact version (`Version`), not a version range (`VersionSpec`). They both resolve to `string`, so the type checker does not catch this yet.

Is there somewhere in the code that is calling `getPeerDependenciesFromRegistry` with a version range instead of an exact version? I think we need to fix it there. Getting the peer dependencies of an entire version range doesn't really make sense to me.

Reply:
You are correct, but for the life of me, I can't remember how I came to find that "issue". I can't reproduce it locally and I can't see how version spec was passed here. So I will close this PR for now